docs/tutorials/odc-stac.rst

```diff
@@ -277,14 +282,19 @@ Type the following into the empty cell below the **Export loaded data** heading:
        )
 
 When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
-You should see seven new files in the file explorer, starting with "LULC_2017.tif" and ending with "LULC_2023.tif".
+You should see seven new files in the file browser, starting with "LULC_2017.tif" and ending with "LULC_2023.tif".
 
 Tutorial complete!
 ------------------
 
 Congratulations, you've used :code:`pystac-client` to search for data in a public STAC catalog and :code:`odc-stac` to load the data into an :code:`xarray`.
-In the last step you exported the loaded data as a series of Cloud Optimised GeoTIFF files, which you can now use in other applications. 
-Make sure you download these files from the file browser before exiting the tutorial space.
+In the last step you exported the loaded data as a series of Cloud Optimised GeoTIFF files, which you can now use in other applications.
+
+.. note::
+   Make sure you download these files from the file browser before exiting the tutorial space as they will be deleted when the tutorial space is closed.
+
+   To download, open the file-browser by clicking the folder icon in the left menu bar.
+   Then, download the "LULC_year.tif" files by right-clicking each file and selecting "Download".
```

Comment:
I think there should be something to indicate there isn't supposed to be a file called `LULC_year.tif`, perhaps call it `LULC_<year>.tif` instead?

Reply:
Thanks, I've updated this.